twitter-core/src/main/java/com/twitter/sdk/android/core/identity/OAuthWebViewClient.java

```diff
@@ -47,6 +48,15 @@ interface Listener {
         this.listener = listener;
     }
 
+    @Override
+    public void onPageStarted(WebView view, String url, Bitmap favicon) {
+        super.onPageStarted(view, url, favicon);
+
+        if (url.equals("https://mix.com/")) {
+            listener.onError(new WebViewException(-1, "OAuth Error", ""));
```

Comment:
Throw a `new TwitterException("bundle incomplete")` to get this treated as an auth cancelation.